SVUI_Maps/SVUI_Maps.xml

```diff
@@ -155,7 +155,7 @@
         </Anchors>
         <Layers>
             <Layer level="ARTWORK">
-                <FontString parentKey="Text" inherits="SVUI_Font_MinimapInfo" text="..." justifyH="RIGHT" justifyV="MIDDLE" setAllPoints="true" />
+                <FontString parentKey="Text" inherits="SVUI_Font_MinimapInfo" text="..." justifyH="CENTER" justifyV="MIDDLE" setAllPoints="true" />
```

Comment:
Updated it to right aligned again. I found where the issue was, it was manually being offset to the right now. We should still watch for long names, but I don't think it will be an issue.

Reply:
Downloaded your fork and ran around. Naigtal looks ok now, hopefully we will be fine. if not we have Github. 

Should be good for merging.